internal/spdxlicense/generate/generate_license_list.go

```diff
@@ -139,9 +175,26 @@ func processSPDXLicense(result LicenseList) map[string]string {
 		}
 	}
 
+	for k, v := range licenseIDs {
+		cleanID := strings.ToLower(v)
+		if remap, exists := overwrite[cleanID]; exists {
+			licenseIDs[k] = remap
+		}
+	}
+
 	return licenseIDs
 }
 
+func findEquivalentLicense(deprecated License, allLicenses []License) *License {
+	for _, l := range allLicenses {
+		if l.isEquivalent(deprecated) {
```

Comment:
we want to return an equivalent license that is not deprecated, this is only guaranteed to return a equivalent license. Should there be another condition here?

Reply:
Good catch, yes I will add the extra condition that it shouldn't be deprecated either. Thank you.